bundles/af-core/src/test/java/com/adobe/cq/forms/core/Utils.java

```diff
@@ -53,15 +53,15 @@
  */
 public class Utils {
 
-    public static InputStream getJson(Object model) {
+    public static InputStream getJson(Object model, Class<? extends Views.Publish> view) {
```

Comment:
had to override this method to send the submission view to test the exporter where json export needs to be done for Submission view